src/features/dashboard/components/AppForm/__tests__/app-form.test.tsx

```diff
@@ -174,7 +174,7 @@ describe('App Form', () => {
     });
 
     const appRedirectUrlInput = screen.getByRole<HTMLInputElement>('textbox', {
-      name: 'Authorization URL (optional)',
+      name: 'Authorization URL (required)',
```

Comment:
We will have to take a second look at the ticket that is created for this. This is because, the authorisation URL is actually optional and is only needed when the user fills in a markup. We don't want to force users in filling an URL when they don't have to (For example, users who want to create a python bot and run it from their desktop). This will really affect user experience.